### DISCUSSION TITLE
How to work with 'Commands' ? (Thermostat)

### DISCUSSION THREAD

**Discussion by guny74** [deconz-community/node-red-contrib-deconz#221] [Q&A] [answered]
Hello,

I connected an thermostat (water floor heating) BHT-002 GA to deconz. 
I am able to get informations from the device.

Now I want to set some values, but do not know how to this with the "commands" option.

Is there any detailed documentation ? (may be I did not find it) 

Thanks and regards
   Thorsten 

(I use the version 2.3.8)

**Comment by Zehir** [comment]
There is no specific UI for that, you will need to use the custom command, set the attribute name and it's value

**Comment by guny74** [comment, marked as answer]
ok, thank you,

How to find the right "attribute path" so set ?

Value from Node after change at the thermostat
![image](https://user-images.githubusercontent.com/30338811/209968033-eb9145f8-5b83-4d1e-8005-3d953a0c0146.png)


So I tried with 
![image](https://user-images.githubusercontent.com/30338811/209968143-ba7d3b14-a0e7-4ced-9c45-aa718429a2a5.png)

but it seams not to work....

Any idea what I do wrong ?

**Reply by guny74** [reply]
I find my error.... I have to sende 18 degrees i hava to send 1800 insted of 18...

Thank you

**Reply by Zehir** [reply]
Yes that is correct. It's depend of the target you select.

If it's in meta.config.heatsetpoint you setup the target as config and the command to heatsetpoint. But you can also set the target as attribute and the command to config.heatsetpoint. It's the same as attribute is the parent of config and state.

**Reply by guny74** [reply]
Thank you
